windows/test-app.js

```diff
@@ -241,23 +219,11 @@ function copyProjectTemplateAndReplace(
     path.join(windowsDir, "ReactTestApp.sln"),
     solutionFileReplacements
   );
-
-  copyAndReplaceAll(
-    path.join(srcRootPath, projDir, "Assets"),
-    destPath,
-    path.join(windowsDir, projDir, "Assets")
-  );
-
-  linkSourceFiles(
-    path.join(srcRootPath, projDir),
-    destPath,
-    path.join(windowsDir, projDir)
-  );
 }
 
 copyProjectTemplateAndReplace(
-  srcRootPath,
-  destPath,
+    destPath,
+    closestNodeModules,
```

Comment:
It looks like your editor is messing with the formatting again. You should run `yarn prettier` to have this file re-formatted.

Reply:
Forgot to run it before pushing. Should be fine now.